### PR TITLE
Remove ^ from staff list in /scav top

### PR DESCRIPTION
**Pull request by PartMan7**
Requested by Litt

### CODE REVIEW

**Comment by PartMan7** [comment]
W-what was that?

**Comment by sparkychildcharlie** [comment]
Room.auth.get returns the higher rank between global and roomauth.  Use room.auth.getDirect instead to get only the room rank

**Comment by PartMan7** [comment]
Ah, I see.

Any idea why it's crashing, though?

**Comment by KrisXV** [comment]
Just resync your branch, `this.parse` is being silly

File: server/chat-plugins/scavengers.ts [hunk 1]
```diff
@@ -1927,7 +1927,7 @@ const ScavengerCommands: ChatCommands = {
 		const ladder = await Leaderboard.visualize('points') as AnyObject[];
 		this.sendReply(
 			`|uhtml${isChange ? 'change' : ''}|scavladder|<div class="ladder" style="overflow-y: scroll; max-height: 300px;"><table style="width: 100%"><tr><th>Rank</th><th>Name</th><th>Points</th></tr>${ladder.map(entry => {
-				const isStaff = room!.auth.has(toID(entry.name));
+				const isStaff = ['+', '%', '@', '*', '#'].includes(room!.auth.getDirect(toID(entry.name)));
```
Comment (mia-pi-git) [inline server/chat-plugins/scavengers.ts:1930]:
```suggestion
				const roomRank = room!.auth.getDirect(toID(entry.name));
				const isStaff = Users.Auth.atLeast(roomRank, '+');
```
Bit cleaner and accounts for other ranks that may be added.

**Comment by Zarel** [comment]
Looks good!

**Pull request merged**